src/test/java/com/wikia/webdriver/testcases/mediatests/modal/VetModalCaptionTests.java

```diff
@@ -34,6 +35,7 @@
 
   @Test(groups = {"VetModalCaption_001"})
   @Execute(asUser = User.USER)
+  @RelatedIssue(issueID = "XW-4586", comment = "broke after update of CKE")
```

Comment:
https://github.com/Wikia/app/pull/14644
We have fix for this in QA, could you check if this test passes after this fix? https://github.com/Wikia/app/pull/14644